vela-templates/definitions/internal/workflowstep/deploy.cue

```diff
@@ -15,15 +15,23 @@ template: {
 		policies:                 parameter.policies
 		parallelism:              parameter.parallelism
 		ignoreTerraformComponent: parameter.ignoreTerraformComponent
+		inlinePolicies:           parameter.inlinePolicies
+	}
+	#Policy: {
+		type:  "topology" | "override" | "replication"
+		name?: string
+		properties: {...}
 	}
 	parameter: {
 		//+usage=If set to false, the workflow will suspend automatically before this step, default to be true.
 		auto: *true | bool
 		//+usage=Declare the policies that used for this deployment. If not specified, the components will be deployed to the hub cluster.
-		policies?: [...string]
+		policies: *[] | [...string]
 		//+usage=Maximum number of concurrent delivered components.
 		parallelism: *5 | int
 		//+usage=If set false, this step will apply the components with the terraform workload.
 		ignoreTerraformComponent: *true | bool
+		//+usage=The policies that embeds in the `deploy` step directly
+		inlinePolicies: *[] | [...#Policy]
```

Comment:
Adding this field for the `deploy` step may not be a good idea. This leads the user to choose between the `policies` and `inlinePolicies` fields. The `policies` field is recommended. 

But, If we want to define an `xx-deploy` step type with some default policies, this field is very useful.

Reply:
I see. So you recommend not exposing inlinePolicies in deploy.cue but in op.#Deploy right?

Reply:
Yes

Reply:
Updated.